roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Install Apache httpd
```

Comment:
We could use `ubi9/httpd-24` but based on the list here we'd have to do some minor customization of it?

Reply:
Normalization of services, configurations mostly.